src/components/dialog-full-screen/dialog-full-screen.js

```diff
@@ -134,14 +134,13 @@ class DialogFullScreen extends Modal {
 
     return (
       <FullScreenHeading>
+        { title }
         <Icon
           className='carbon-dialog-full-screen__close'
           data-element='close'
           onClick={ this.props.onCancel }
           type='close'
         />
-
-        { title }
```

Comment:
what was the reason for this change? does it effect the UI at all?

Reply:
The icon has to be above the title bar. I couldn't see why this worked before tbh.